stable/nginx-ingress/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -45,6 +45,8 @@ spec:
             - --host={{ template "nginx-ingress.controller.fullname" . }}-admission,{{ template "nginx-ingress.controller.fullname" . }}-admission.{{ .Release.Namespace }}.svc
             - --namespace={{ .Release.Namespace }}
             - --secret-name={{ template "nginx-ingress.fullname". }}-admission
+          resources:
```

Comment:
lets put this into a `with` block so that when the values of the resources are empty this doesn't get filled in